ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -225,20 +278,37 @@ export function AlertModal({
           borderWidth={1}
           display={headerStartAccessory ? Display.InlineFlex : Display.Block}
         />
-        <AlertHeader selectedAlert={selectedAlert} />
+        <AlertHeader
+          selectedAlert={selectedAlert}
+          customAlertTitle={customAlertTitle}
+        />
```

Comment:
The current implementation has the close button above a divider line though the Figma version has no divider line and the UI appears closer to the close button

Figma
![CleanShot 2024-05-21 at 02 19 19](https://github.com/MetaMask/metamask-extension/assets/20778143/9e42f461-e572-477c-9469-3c81b27011cd)

![CleanShot 2024-05-21 at 02 19 35](https://github.com/MetaMask/metamask-extension/assets/20778143/a96502da-6a6c-4784-b5df-1e40e525ec79)

Reply:
That is a good point I asked Sayalee because there are some inconsistencies in the figma and it makes sense to have a clear separation of the header and the content.
In any case, I have reduced the distance here already, and depending on Syalee's feedback I will push it to a subsequent PR.